src/Microsoft.IdentityModel.Tokens/BaseConfigurationManager.cs

```diff
@@ -63,6 +86,18 @@ public virtual Task<BaseConfiguration> GetBaseConfigurationAsync(CancellationTok
             throw new NotImplementedException();
         }
 
+        /// <summary>
+        /// Gets all valid last known good configurations.
+        /// </summary>
+        /// <returns>A collection of all valid last known good configurations.</returns>
+        internal ICollection<BaseConfiguration> GetValidLkgConfiguraitons()
```

Comment:
Please rename: GetValidLkgConfiguraitons => GetValidLkgConfigurations.

'i' is in the wrong place.